packages/preview-middleware/templates/test/qunit.js

```diff
@@ -11,18 +11,21 @@ sap.ui.loader.config({
         },
         "sap/ui/qunit/sinon-qunit-bridge": {
             deps: ["sap/ui/thirdparty/qunit", "sap/ui/thirdparty/sinon-4"]
-        }        
+        }
     }
 });
 
+window.QUnit ??= {};
```

Comment:
BTW: wouldn't it look nicer this way:

```js
window.QUnit = Object.assign({}, { config: { autostart: false } }, window.QUnit);
```

Reply:
Good idea but what's the need for source1 = window.QUnit? Isn't this the same as

```js
window.QUnit = Object.assign({}, { config: { autostart: false } });
```

🤔

Reply:
I think I understood your intention. It's to ensure that is does not override QUnit in case QUnit already exists (so this would not be a preconfig). Right?